doc/build-openbsd.md

```diff
@@ -46,15 +44,16 @@ from ports. However you can build it yourself, [using depends](/depends).
 Refer to [depends/README.md](/depends/README.md) for detailed instructions.
 
 ```bash
+pkg_add bash gmake gtar
```

Comment:
This should be moved to the depends readme, not duplicated here.

Reply:
Thanks! Done.